corehq/apps/export/static/export/js/download_export.ng.js

```diff
@@ -286,6 +287,21 @@
                             hqImport('export/js/utils').capitalize(exportDownloadService.exportType), "Saved");
             hqImport('analytix/js/kissmetrix').track.event("Clicked Download button");
         };
+
+        $scope.sendEmailUponCompletion = function () {
+            setTimeout(function () {
```

Comment:
I would add a comment here about why you're doing this in an asynchronous loop. It may not be clear to the reader that it's to wait until `download_id` is available in the scope.